translations/en-us.yaml

```diff
@@ -2739,6 +2775,59 @@ modalAboutComponent:
   machineService: Machine Service
   close: Close
 
+modalAddPayment:
+  header: Add New Payment Method
+  save: Save
+  errors:
+    cc: Invalid credit card number
+    exp: Invalid Expiry
+    cvc: Invalid CVC number
+    street: A street is required.
+    city: A city is required.
+    state: A state is required.
+    zip: A zipcode is required.
+  form:
+    currency:
+      label: Select your currency
+    cc:
+      label: Credit Card Number
+      placeholder: '•••• •••• •••• ••••'
+    default:
+      label: Default Card
+    name:
+      label: Name On Credit Card
+      placeholder: 'John Smith'
+    exp:
+      label: Expiration
+      placeholder: '••/••'
+    cvc:
+      label: CVC
+      placeholder: '•••'
+    address:
+      addressLine1:
+        label: Address
+        placeholder: 123 Some Street East
```

Comment:
Clearly this should be [742 Evergreen Terrace, Springfield OR](http://www.cbsnews.com/news/the-simpsons-creator-matt-groening-reveals-location-of-springfield/)